Atari-Game-Player/DQN/model.py

```diff
@@ -0,0 +1,718 @@
+import random
+import numpy as np
+import numpy as np
+import cv2
+
+import random
+import os
+import json
+import time
+
+import gym
+
+env = gym.envs.make("BreakoutDeterministic-v4")
+
+import tensorflow as tf
+from tensorflow.keras.initializers import VarianceScaling
+from tensorflow.keras.layers import (
+    Add,
+    Conv2D,
+    Dense,
+    Flatten,
+    Input,
+    Lambda,
+    Subtract,
+)
+from tensorflow.keras.models import Model
+from tensorflow.keras.optimizers import Adam, RMSprop
+
+from typing import Any, Type, Union, List
+import pickle
+from tensorflow import keras
+
+
+def process_frame(frame, shape=(84, 84)):

```

Comment:
Typing could make a lot of sense here so that anyone looking at the code understands what a frame actually is

Reply:
Ok Sure